lib/example-config.yml

```diff
@@ -0,0 +1,7 @@
+sensor: landsat8
```

Comment:
If we're going to have the actual config checked in, then do we still need an example config?